.bazelrc

```diff
@@ -147,6 +145,15 @@ build:auto-release --config=remote
 build:auto-release --remote_instance_name=buildbuddy-io/buildbuddy/auto-release
 build:auto-release -c opt
 
+build:linux --incompatible_enable_cc_toolchain_resolution
+build:linux --action_env BAZEL_DO_NOT_DETECT_CPP_TOOLCHAIN=1
+build:linux --@io_bazel_rules_go//go/config:static=true
+# build:linux --@io_bazel_rules_go//go/config:linkmode=pie
+build:linux --copt=-fPIC
+# build:linux --linkopt=-static
+# build:linux --linkopt=-static-pie
+# build:linux --action_env=GOTAGS=-tags=osusergo,netgo
```

Comment:
can we remove the commented lines here if they're not necessary?

Reply:
yup, fine with me, I just left them so if I go back and do pie stuff I remember the flags needed for it.